cdap-client/src/main/java/io/cdap/cdap/client/ApplicationClient.java

```diff
@@ -438,9 +444,9 @@ public void deploy(NamespaceId namespace, File jarFile,
   /**
    * Creates an application with a version using an existing artifact.
    *
-   * @param appId the id of the application to add
```

Comment:
here and other places same.

Reply:
Seems like the whole file got formatted, will resubmit this file.

Reply:
Done